docs/en/latest/deployment-modes.md

```diff
@@ -70,20 +71,20 @@ The instance of APISIX deployed as the traditional role will:
 
 ## Decoupled
 
-In the decoupled deployment mode the data plane and control plane instances of APISIX are deployed separately. i.e one instance of APISIX is configured to be a data plane and the other to be a control plane.
+In the decoupled deployment mode the `data_plane` and `control_plane` instances of APISIX are deployed separately, i.e one instance of APISIX is configured to be a *data plane* and the other to be a *control plane*.
```

Comment:
In the decoupled deployment mode the `data_plane` and `control_plane` instances of APISIX are deployed separately, _i.e._, one instance of APISIX is configured to be a *data plane* and the other to be a *control plane*.

Reply:
@nfrankel I'm sorry, but can you please highlight the required changes? I'm not able to see it. 😅
Also, I did use Grammarly to rectify most of the grammatical mistakes, so I think it should be fine now.😄

Reply:
There should be a command before and after "i.e." (and "e.g."). Grammarly is adamant about it.

Reply:
Ah, I see now. Just rectified it, can you have a look at whether it's ready to merge?